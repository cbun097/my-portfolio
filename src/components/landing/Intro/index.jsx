import React from 'react'
import AnchorLink from 'react-anchor-link-smooth-scroll'
import { Header } from 'components/theme'
import { Container, Button } from 'components/common'
import dev from 'assets/illustrations/dev-focus.svg'
import { Wrapper, IntroWrapper, Details, Thumbnail } from './styles'

export const Intro = () => (
  <Wrapper>
    <Header />
    <IntroWrapper as={Container}>
      <Details>
        <h1>Hi There!</h1>
        <h4>I’m Claire and I’m a front-end developer & student!</h4>
        <Button purple>
          <a
            href="https://www.linkedin.com/in/claire-bun-3678ba132/"
            rel="noopener noreferrer"
            target="_blank"
          >
            View my resume
          </a>
        </Button>
        <Button as={AnchorLink} href="#contact">
          Hire me
        </Button>
      </Details>
      <Thumbnail>
        <img
          src={dev}
          alt="I’m Claire and I’m a front-end developer & student!"
        />
      </Thumbnail>
    </IntroWrapper>
  </Wrapper>
)
