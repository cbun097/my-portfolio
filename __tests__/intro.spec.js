import React from 'react'
import renderer from 'react-test-renderer'
import { Intro } from '../src/components/landing/Intro/index'

describe('Intro Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Intro />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
