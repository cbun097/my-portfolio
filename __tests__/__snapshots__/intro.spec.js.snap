// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro Component renders correctly 1`] = `
<div
  className="sc-fzqBZW diFehN"
>
  <div
    className="sc-fzpans hacvFS"
  >
    <div
      className="sc-fzplWN hCvsbW"
      onClick={[Function]}
    />
    <div
      className="sc-AxjAm dOpVwC sc-Axmtr ijfsfV"
    >
      <a
        aria-current="page"
        className=""
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={Object {}}
      >
        Claire Bun
      </a>
      <div
        className="sc-AxheI kdPQrW"
      >
        <a
          href="#about"
          onClick={[Function]}
        >
          About
        </a>
        <a
          href="#projects"
          onClick={[Function]}
        >
          Projects
        </a>
        <a
          href="#blog"
          onClick={[Function]}
        >
          Blog
        </a>
        <a
          href="#contact"
          onClick={[Function]}
        >
          Contact
        </a>
      </div>
    </div>
    <div
      className="sc-AxmLO kTyflp"
      onClick={[Function]}
    >
      <div
        className="sc-fzozJi kVfXSm"
      />
      <div
        className="sc-fzozJi kVfXSm"
      />
      <div
        className="sc-fzozJi kVfXSm"
      />
    </div>
    <div
      className="sc-fzoLsD jiOLuR"
      onClick={[Function]}
    >
      <div
        className="sc-AxheI cJfRhu"
      >
        <a
          href="#about"
          onClick={[Function]}
        >
          About
        </a>
        <a
          href="#projects"
          onClick={[Function]}
        >
          Projects
        </a>
        <a
          href="#blog"
          onClick={[Function]}
        >
          Blog
        </a>
        <a
          href="#contact"
          onClick={[Function]}
        >
          Contact
        </a>
      </div>
    </div>
  </div>
  <div
    className="sc-AxjAm dOpVwC sc-fzqNJr PzOZk"
  >
    <div
      className="sc-fzoyAV jtxeRq"
    >
      <h1>
        Hi There!
      </h1>
      <h4>
        I’m Claire and I’m a front-end developer & student!
      </h4>
      <button
        className="sc-AxirZ cyWyrQ"
      >
        <a
          href="https://www.linkedin.com/in/claire-bun-3678ba132/"
          rel="noopener noreferrer"
          target="_blank"
        >
          View my resume
        </a>
      </button>
      <a
        className="sc-AxirZ ewUoIv"
        href="#contact"
        onClick={[Function]}
      >
        Hire me
      </a>
    </div>
    <div
      className="sc-fzoLag boaARM"
    >
      <img
        alt="I’m Claire and I’m a front-end developer & student!"
        src="test-file-stub"
      />
    </div>
  </div>
</div>
`;
